src/components/header.tsx

```diff
@@ -105,8 +105,10 @@ export const Header = observer(
               <Logo mode="white" className="mr-4" />
 
               <span className="text-grey-blue">
-                Anfisa front: {process.env.REACT_APP_VERSION} back:{' '}
-                {toJS(dirinfoStore.dirinfo).version as string}
+                {t('header.version', {
+                  frontend: process.env.REACT_APP_VERSION,
+                  backend: toJS(dirinfoStore.dirinfo).version,
+                })}
```

Comment:
According to the [task](https://quantori.atlassian.net/browse/FOROME-588), they want the versions to be displayed in a column. In your version they are displayed in a row.

Reply:
Changed the layout to columns

Reply:
They still displayed in a row.
Can you display them like this? And give them font-size 13px, in order not to bring too much attention to them
<img width="376" alt="Screenshot 2022-01-31 at 23 21 09" src="https://user-images.githubusercontent.com/96476703/151867232-e151f260-8866-4f9b-8cdc-95d8ec75dee0.png">

Reply:
![image](https://user-images.githubusercontent.com/98574721/151912858-e543e992-4eca-4b71-ab57-26540d48c390.png)

Right now, it displayed in a column